### DISCUSSION TITLE
Error when compiling EthanC's Jekyll (Call of Duty asset exporter)

### DISCUSSION THREAD

**Discussion by mrmaller1905** [community/community#123426] [Programming Help]
### Body

I tried to compile Jekyll (Call of Duty asset exporter) by EthanC, but I get:
```
1>------ Build started: Project: PhilLibX, Configuration: Debug Any CPU ------
1>C:\Users\PC\Source\Repos\Jekyll\PhilLibX\IO\ProcessReader.cs(193,32,193,42): error CS0103: The name 'Decryption' does not exist in the current context
1>Done building project "PhilLibX.csproj" -- FAILED.
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========
```
Can someone help me fix this error?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Hey @mrmaller1905! Were you able to solve your issue? It would be great to reference if folks have this issue in the future.

No worries if not, hopefully someone else has encountered this before!
